packages/grafana-ui/src/components/PanelChrome/PanelChrome.story.tsx

```diff
@@ -105,4 +108,34 @@ export const Examples = () => {
   );
 };
 
+export const Basic: ComponentStory<typeof PanelChrome> = (args: PanelChromeProps) => <PanelChrome {...args} />;
+
+const LoadingIcon = <PanelChrome.LoadingIndicator loading onCancel={() => {}} />;
+const ErrorIcon = <PanelChrome.ErrorIndicator error="Error text" onClick={() => {}} />;
+
+const leftItems = { LoadingIcon, ErrorIcon };
+
+Basic.argTypes = {
+  leftItems: {
+    options: Object.keys(leftItems),
+    mapping: leftItems,
+    control: {
+      type: 'select',
+      labels: {
+        LoadingIcon: 'With loading icon',
+        ErrorIcon: 'With error icon',
+      },
+    },
+  },
+};
```

Comment:
This looks so cool! 🎉  Can we add an option to clear the icon as well? Not sure if the storybook select control lets us do that but alternatively we can add another option to the select for "With no icons"?

Reply:
Thanks for the feedback, landed in [0050d8a](https://github.com/grafana/grafana/pull/54741/commits/0050d8ada22d2a092a721a86254ad92a496f367d), I have added one more option to the select.